lib/publishing_api_payload.rb

```diff
@@ -112,4 +116,23 @@ def roles_and_people(role_appointments)
         memo[:people] = (memo[:people] + people).uniq
       end
   end
+
+  def attachments
+    file_attachments = edition.file_attachment_revisions
+
+    file_attachments.map do |attachment|
+      FileAttachment.new(attachment, edition.document).payload
```

Comment:
I wonder if we want to call this FileAttachmentPayload so it feels more distinct in namespace from the global FileAttachment model?